### ISSUE TITLE
USER EDIT COLOR BUTTONS

### ISSUE THREAD

**Issue by zomer-g** [OriHoch/data4dappl#75]
![image](https://user-images.githubusercontent.com/30901226/45253038-13355380-b369-11e8-9286-80651a33e041.png)

https://www.odata.org.il/user/edit/zomerg

the red should be #CD352D instead of #D8554E
the orange should be #AA660E instead of #F1AB52
the blue should stay

**Comment by OriHoch** [comment]
done + deployed
